libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -418,7 +410,7 @@ export class SolanaWeb3Manager {
 
       // Multiply by 10^10 to maintain same decimals as eth $AUDIO
       const decimals = AUDIO_DECMIALS - WAUDIO_DECMIALS
-      return tokenAccount.amount.mul(Utils.toBN('1'.padEnd(decimals + 1, '0')))
+      return tokenAccount.amount * BigInt('1'.padEnd(decimals + 1, '0'))
```

Comment:
this seems like it could incur precision loss. is `*` reimplmented on bigint?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt

answered my own Q. cool. TIL. let's kill BN.js...

Reply:
believe so: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#operators